openwisp_notifications/widgets.py

```diff
@@ -0,0 +1,39 @@
+from pydoc import locate
+
+from openwisp_notifications import settings as app_settings
+
+
+class IgnoreObjectNotificationWidgetMedia:
+    extend = True
+    js = [
+        'admin/js/jquery.init.js',
+        'openwisp-notifications/js/object-notifications.js',
+    ]
+    css = {'all': ['openwisp-notifications/css/object-notifications.css']}
+
+
+def add_object_notification_widget():
```

Comment:
this function is a bit magical. I would flag it as private (rename it to `_add_object_notification_widget`) so we can change it without worrying too much about backward compatibility in the future if we need it